src/redux/store.ts

```diff
@@ -2,5 +2,6 @@ import { applyMiddleware, compose, legacy_createStore as createStore } from 'red
 import thunkMiddleware from 'redux-thunk';
 import { rootReducer } from './reducers/reducers';
 
+// @ts-ignore
```

Comment:
MAy be helpful =)
https://stackoverflow.com/questions/12709074/how-do-you-explicitly-set-a-new-property-on-window-in-typescript

Reply:
Thanks, this was helpful :)
Done!